components/Image /image.css

```diff
@@ -0,0 +1,21 @@
+/* global css sheet for fonts and icons */
+@import '/css/main.css';
+
+/* global image HTML5 tag */
+img {
+    width: 600;
+    height: 400;
+    margin: 1rem;
```

Comment:
Here you should not specify the height and width of the image, you should specify the height and width of the image container and  the image should scale to that container